desktop-app/app/reducers/browser.js

```diff
@@ -195,9 +197,17 @@ function _getActiveDevices() {
 }
 
 function _getUserPreferences(): UserPreferenceType {
-  return (
-    settings.get(USER_PREFERENCES) || {removeFixedPositionedElements: true}
-  );
+  const fromSettings = settings.get(USER_PREFERENCES);
+  if (fromSettings == null)
+    return {
+      removeFixedPositionedElements: true,
+      permissionManagement: PERMISSION_MANAGEMENT_OPTIONS.ALLOW_ALWAYS,
+    };
+
+  if (fromSettings.permissionManagement == null)
+    fromSettings.permissionManagement = PERMISSION_MANAGEMENT_OPTIONS.ALLOW_ALWAYS;
```

Comment:
@jjavierdguezas Can you set this as the default value in the https://github.com/responsively-org/responsively-app/blob/master/desktop-app/app/settings/migration.js#L25 file? 

So that we can avoid this fallback cases.

Reply:
done